CommonTools/ParticleFlow/interface/TopProjectorAlgo.h

```diff
@@ -17,6 +17,7 @@
 
 #include "DataFormats/ParticleFlowCandidate/interface/PFCandidateFwd.h"
 
+#include "DataFormats/Candidate/interface/Candidate.h"
 #include "DataFormats/Candidate/interface/CandidateFwd.h"
```

Comment:
similar to #18805 and #21415 the solution should be to include the Candidate.h in CandidateFwd.h and then move CandidateFwd.h to CandidateDefs.h